### DISCUSSION TITLE
咨询和建议

### DISCUSSION THREAD

**Discussion by yx3016** [lsky-org/lsky-pro#720] [想法]
1、请问 ，是否有官方标准的 docker 镜像。便于我们这些小白部署[lsky-pro]
2、建议，是否可以增加，与企业微信绑定的功能，方便部署后，集成到企业内部，便于使用。否则还要注册用户，有些繁琐。

**Comment by 0xxb** [comment]
没有标准版 docker 镜像，建议使用社区小伙伴制作的镜像。#321
暂不考虑企业微信绑定功能。
